test/webserv/integration/test_http_get.py

```
@@ -1,37 +1,11 @@
-try:
-    # client_module をビルド
-    build_client_module()
-    # client_module.client をインポート
-    import client_module.client as client
-except subprocess.CalledProcessError as e:
-    print(f"Build failed: {e}")
-except ImportError as e:
-    print(f"Import failed: {e}")
-
+from common_functions import send_request_and_assert_response
+from common_response import *
```

Ruff

`from common_response import *` used; unable to detect undefined names

(F403)